### DISCUSSION TITLE
Add a Gemini as llm

### DISCUSSION THREAD

**Discussion by GeorgeMyller** [strnad/CrewAI-Studio#2] [Ideas]
Can you put a Gemini (API) models as llm to run tools and agents?

**Comment by strnad** [comment]
thanks for suggestion... google ai added :) but it will probably still require openai for embeddings, i will try to implement more embeddings options later, but i am not sure if we can force all tools to use custom embeddings.

**Comment by strnad** [comment]
https://github.com/joaomdmoura/crewAI/issues/719

**Comment by strnad** [comment]
i see in docs that for RAGtool based tools it should be possible to set the custom model, i will try to do it when i am home, I'm quite busy right now

**Comment by analyticsrepo01** [comment]
Does the Manager LLM use Vertex AI too? 
isnt the backend similar to how crew AI , so can we use vertex AI and embeddings similar to crew AI.
